packages/java-shell/src/test/js/run-tests.ts

```diff
@@ -4,16 +4,12 @@ import fs from 'fs';
 import path from 'path';
 import { startTestServer } from '../../../../../testing/integration-testing-hooks';
 
-const port = '27019';
-const authlessUri = `mongodb://localhost:${port}`;
-const uri = `mongodb://admin:admin@localhost:${port}`;
-
 describe('java-shell tests', function() {
-  this.timeout(300_000);
+  this.timeout(600_000);
```

Comment:
Feels like 5 minutes should have been enough, but this timed out once, then I doubled it, then it didn’t time out, soooooo … yeah :woman_shrugging: